src/test/java/io/kabanero/selenium/InstanceIT.java

```diff
@@ -2,52 +2,101 @@
+        options.setHeadless(true);
+        driver = new ChromeDriver(options);
     }
 
     @Before
-    public void beforeTest() throws IOException {
-        // selenium stuff
-        ChromeOptions options = new ChromeOptions();
-        options.addArguments("--ignore-certificate-errors");
-        options.setHeadless(true);
-		driver = new ChromeDriver(options);
+    public void beforeTest() {
+        driver.get(baseUrl);
+        js = (JavascriptExecutor) driver;
+        driver.manage().timeouts().setScriptTimeout(5, TimeUnit.SECONDS);
     }
 
-    @After
-    public void teardown() {
+    @AfterClass
+    public static void teardown() {
         if (driver != null) {
             driver.quit();
         }
     }
 
     @Test
-    public void hasCorrectTitleIT(){
-        String baseUrl = "https://localhost:9443/instance/";
-
-        driver.get(baseUrl);
+    public void hasCorrectTitleIT() {
         String expectedTitle = "Kabanero";
         String actualTitle = driver.getTitle();
+
         assertEquals("title equals " + expectedTitle, expectedTitle, actualTitle);
     }
+
+    @Test
+    public void displaysCorrectInstanceNameIT() throws InterruptedException {
+        // execute javascript function setInstanceSelections() and pass it mock data to test against
+        js.executeScript("setInstanceSelections(" + fakeInstancesEnpointData + ")");
+        String expectedInstanceName = "kabanero";
+        String actualInstanceName = driver.findElement(By.id("instance-accordion"))
+                .findElement(By.className("bx--accordion__title")).getText();
+        assertEquals("instance name equals " + expectedInstanceName, expectedInstanceName, actualInstanceName);
+    }
+
+    @Test
+    public void copiesToClipBoardIT() throws InterruptedException, UnsupportedFlavorException, IOException {
+        // execute javascript function loadAllInfo() and pass it mock data to test against
+        js.executeScript("setInstanceCard(" + fakeKabaneroInstanceData + ")");
+
+        String expectedAppsodyURL = "https://github.com/kabanero-io/collections/releases/download/0.5.0/kabanero-index.yaml";
+        String actualAppsodyURL = getClipboardValue(By.className("appsody-url"));
+        assertEquals("instance name equals " + expectedAppsodyURL, expectedAppsodyURL, actualAppsodyURL);
+
+        String expectedCodewindURL = "https://github.com/kabanero-io/collections/releases/download/0.5.0/kabanero-index.json";
+        String actualCodewindURL = getClipboardValue(By.className("codewind-url"));
+        assertEquals("instance name equals " + expectedCodewindURL, expectedCodewindURL, actualCodewindURL);
+
+        String expectedCliURL = "kabanero-cli-kabanero.apps.alohr.os.fyre.ibm.com";
+        String actualCliURL = getClipboardValue(By.id("management-cli"));
+        assertEquals("instance name equals " + expectedCliURL, expectedCliURL, actualCliURL);
+    }
```

Comment:
great test! I think we should separate them out into 4 tests (so if one fails we know exactly which one fails) I added 2 more tests for the copy icon too

Make sure to name the tests accordingly as well.
1. Click the Appsody URL input box and test clipboard value
1. Click the Appsody Copy icon and test clipboard value
1. Click the Codewind URL input box and test clipboard value
1. Click the Codewind copy icon and test clipboard value